### PR TITLE
Replace Sci-Hub X Now's chrome extension link with their github's

### PR DESCRIPTION
**Pull request by lumikalt**
Previously the website listed only the chrome extension, but their Github repo lists also the firefox extension and the edge extension, for whoever needs them.

### CODE REVIEW

**Comment by kekkodance** [comment]
you can remove the [Github] tag, it's useless

File: Literature/e-Books.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ description: Reliable sources to obtain e-Books.
 # Scientific Journals/Technical Manuals
 
 - [**Sci-Hub (.se)**](https://sci-hub.se/) | [**.st**](https://sci-hub.st/) | [**.ru**](https://sci-hub.ru/)    
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Sci-Hub X Now!](https://chrome.google.com/webstore/detail/sci-hub-x-now/gmmnidkpkgiohfdoenhpghbilmeeagjj) (Chromium) - Opens the sci-hub page for the article you want to read.  
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Sci-Hub X Now!](https://github.com/gchenfc/sci-hub-now) [Github]
```
Comment (Helmasko) [inline Literature/e-Books.md]: any specific reason you removed the description?

Comment (lumikalt) [reply Literature/e-Books.md]: if by description you mean [Github], it was because of the comment above.

Comment (kekkodance) [reply Literature/e-Books.md]: you can keep the extension description but remove [Github].

**Comment by Helmasko** [comment]
I'm assuming you accidentally removed the desc "Opens the sci-hub page for the article you want to read." so I'l just re-add it and merge.
Thank you for the contribution!

File: Literature/e-Books.md (hunk 1 above)
Comment (lumikalt) [reply Literature/e-Books.md]: Oh i completelly missed the fact that i removed that aaa
just used vim's d$ without thinking :skull:

**Comment by lumikalt** [comment]
Yes thanks I didn't notice it :skull:

**Pull request merged**